particle.inf

```diff
@@ -38,6 +38,12 @@ CatalogFile=particle.cat
 ; Core
 %DEVICE_DESCRIPTION_CORE%=DriverInstall, USB\VID_1D50&PID_607D
 
+; Photon pre-0.6.0
+%DEVICE_DESCRIPTION_PHOTON_3%=DriverInstall, USB\VID_2B04&PID_C006&REV_0200
```

Comment:
The `MI_00` id only appears for composite devices. Only 0.6.0 support composite USB, so `&MI_00` entry will not match pre-0.6.0 devices.